synapse/lib/layer.py

```diff
@@ -3338,6 +3338,33 @@ async def liftByDataName(self, name):
             sode['nodedata'] = {name: s_msgpack.un(byts)}
             yield None, buid, sode
 
+    async def setStorNodeProp(self, buid, prop, valu):
+        newp = self.core.model.reqProp(prop)
+
+        newp_valu = newp.type.norm(valu)[0]
+        newp_name = newp.name
+        newp_stortype = newp.type.stortype
+        newp_formname = newp.form.name
+
+        set_edit = (EDIT_PROP_SET, (newp_name, newp_valu, None, newp_stortype), ())
+        nodeedits = [(buid, newp_formname, [set_edit])]
+
+        _, changes = await self.saveNodeEdits(nodeedits, {'time': s_common.now()})
```

Comment:
Should we include a 'user' in the nodeedits meta? The main use case for this is migrations so it will usually just be the root user but maybe we should still include it?